packages/react-aria-components/scripts/create-react-aria-components/templates/vanilla-css/components/Button/Button.css

```diff
@@ -0,0 +1,52 @@
+.react-aria-Button {
+  --border-color: var(--spectrum-alias-border-color);
```

Comment:
what's providing the spectrum variables?

Reply:
Definitely need to clean up the templates. These were copied over from the docs.